packages/react/src/components/controls/StartAudio.tsx

```diff
@@ -5,35 +5,32 @@ import { useRoomContext } from '../../context';
 import { useObservableState } from '../../hooks/internal/useObservableState';
 import { mergeProps } from '../../utils';
 
-interface UseStartAudioProps {
-  room: Room;
-  props: React.ButtonHTMLAttributes<HTMLButtonElement>;
-}
-
-export function useStartAudio({ room, props }: UseStartAudioProps) {
-  const { className, roomAudioPlaybackAllowedObservable, handleStartAudioPlayback } = React.useMemo(
+/**
+ * In many browser to start audio playback, the user must perform a user-initiated event such as clicking a button.
+ * The `useStatAudio` hook returns an object with a boolean `canPlayAudio` flag
+ * that indicates whether audio playback is allowed in the current context,
+ * as well as a `startAudio` function that can be called in a button `onClick` callback to start audio playback in the current context.
+ *
+ * @see Autoplay policy on MDN web docs for more info: {@link https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API/Best_practices#autoplay_policy}
+ * @alpha
+ */
+export function useStartAudio(room: Room) {
+  const { roomAudioPlaybackAllowedObservable, handleStartAudioPlayback } = React.useMemo(
```

Comment:
it's a bit unfortunate that both (this hook and the `StartAudio` button) call the same function `setupStartAudio`. Wondering if we can avoid that.

Reply:
The alternative was to hardcode the class name. We could have a gernall hook without the class name and a useStartAudio that wraps the gernall hook, merges props and adds the class name.
Or do you have another suggestion?

Reply:
I like the approach of splitting it in two! but that would also mean changing core API right?

Reply:
As it turned out, no change to the core API was necessary.